### ISSUE TITLE
All alerts should be announced by screen readers

### ISSUE THREAD

**Issue by plannero** [umbraco/Umbraco-CMS#14521]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.4.0

### Bug summary

Some, but not all, confirmation messages are announced by screen readers. 

### Specifics

Confirmation message after moving a page is not presented to screen reader users. Should be put in an `aria-live` area so that screen readers announce them even if user has focus somewhere else on the screen. The message is in a div with `class="alert"`, which I presume is what brings the nice green color. An aria-live attribute (or equivalent) could be added alongside with the class attribute, or on the parent div, if this container element is always present. 

<img width="379" alt="alert" src="https://github.com/umbraco/Umbraco-CMS/assets/2157085/b5be7f88-4280-41a6-87e6-da4228736cf5">
<img width="574" alt="green" src="https://github.com/umbraco/Umbraco-CMS/assets/2157085/7621a830-8ba2-41c5-9ca7-5846eda29ed6">

### Steps to reproduce

Move a page while using a screen reader. Listen for the confirmation. You will hear none, unless you listen through all positions on the page where you suspect it may be present.

### Expected result / actual result

The green confirmation should be announced without any need to search for it.

---
_This item has been added to our backlog AB#62241_

**Comment by andr317c** [comment]
Hey!
Thanks a lot for the time to post this issue. This is something we would like help with. I will mark the issue as up for grabs! 😄
